### PR TITLE
Move responsiveness code from okta-core to okta-theme in signin-widget.

### PR DESCRIPTION
**Pull request by santhoshbalakrishnan**
## Description

Styles are being moved from https://github.com/okta/okta-core/blob/master/clients/loginpage/sass/loginpage-theme.scss#L43

## Reviewers

- @rchild-okta 
- @mauriciocastillosilva-okta 

## Screenshare

siw-resp

### CODE REVIEW

**Comment by bretterer** [comment]
Any way to make this so the logo remains in the center?

**Comment by santhoshbalakrishnan** [comment]
@brianretterer-okta Good thing to think about! We do support that as long as the logo is of supported size. This is a bit more than that, why is why it didn't work. Also, it started breaking only below the size of iphone 4, which I do not know if we still support. However, that is a good thing to add. But, it's a little bit out of the scope of the current PR, where we concentrated on the widget itself aligning proper on okta-theme.

File: assets/sass/okta-theme.scss [hunk 1]
```diff
@@ -820,3 +820,33 @@
...
+/*  Remove the background image and borders on a smaller window size */
+@include max-width-mq(600px) {
+  // scss-lint:disable ImportantRule
+  .login-bg-image {
+    background-image: none !important;
+    background-color: #fff !important;
+    filter: unset !important;
+  }
+
+  /* See line 59. Unfortunately, we have to use the selector this way to override at small screen sizes. */
```
Comment (mauriciocastillosilva-okta) [inline assets/sass/okta-theme.scss]: nit: Might be just enough to say we use the same selector than in line 59?

Comment (santhoshbalakrishnan) [reply assets/sass/okta-theme.scss]: Haha okay!

Comment (santhoshbalakrishnan) [reply assets/sass/okta-theme.scss]: Changed to this: Using the same selector as line 59 to override styles at small screen sizes.

Comment (mauriciocastillosilva-okta) [reply assets/sass/okta-theme.scss]: sounds good. Once you push your changes this is ready, but let me know if you need me to review again.

**Pull request merged**